sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BaseTrafoPwPanel
+
+
+class AggrDist(BaseTrafoPwPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
+        example: AggrDist(ScipyDist()) is mean Euclidean distance between series
+
+    Components
+    ----------
+    transformer: pairwise transformer of BaseTrafoPw scitype (tabular pairwise)
+
+    Hyper-parameters
+    ----------------
+    aggfun: aggregation function 2D np.array -> float
```

Comment:
```suggestion
    aggfunc: aggregation function 2D np.array -> float
```
Only because aggfunc is used elsewhere in sktime, e.g. https://github.com/alan-turing-institute/sktime/blob/main/sktime/forecasting/compose/_ensemble.py

Reply:
ah, no fun(c)

Reply:
You should compete in the UK pun championships: https://www.youtube.com/watch?v=hriVI0x_OEc

Reply:
I think I already won without participating.

Reply:
You first need a competitor name, which must be a pun containing 'pun', before you can even claim to have won the contest

Reply:
fine. How about "puncipal data scientist"?

Also, one of the greatest python packages was created when a larger number of telepaths met to play computer games: they formed a psychic LAN.